cirq/ops/raw_types.py

```diff
@@ -319,6 +321,17 @@ def _qid_shape_(self) -> Tuple[int, ...]:
         (3, 3) for a 2-qutrit ternary gate.
         """
 
+    def _commutes_with_(self, other: Any, atol: Union[int, float] = 1e-8
```

Comment:
I forget why we picked `_commutes_with_` instead of just `_commutes_`.

Reply:
Went with `_commutes_` everywhere.